ArcGISRuntimeSDKQt_CppSamples/AR/DisplayScenesInTabletopAR/PermissionsHelper.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 ESRI
+//
+// All rights reserved under the copyright laws of the United States
+// and applicable international laws, treaties, and conventions.
+//
+// You may freely redistribute and use this sample code, with or
+// without modification, provided you include the original copyright
+// notice and use restrictions.
+//
+// See the Sample code usage restrictions document for further information.
```

Comment:
Does this file require the apache license or is this one OK? Same comment for all PermissionsHelper source/header files.

Reply:
You are most definitely right!